setup.py

```diff
@@ -30,7 +30,7 @@
     packages=find_packages(exclude=['tests*']),
     package_data={'awscli': ['data/*.json', 'examples/*/*.rst',
                              'examples/*/*/*.rst', 'topics/*.rst',
-                             'topics/*.json']},
+                             'topics/*.json', 'bin/aws_bash_completer']},
```

Comment:
It probably does make more sense as package data, but to be consistent with the other completers , I'll just go ahead and switch this over to scripts/.